java/bench/pom.xml

```diff
@@ -81,13 +81,7 @@
       <dependency>
         <groupId>io.airlift</groupId>
         <artifactId>aircompressor</artifactId>
-        <version>0.10</version>
-        <exclusions>
-          <exclusion>
-            <groupId>io.airlift</groupId>
-            <artifactId>slice</artifactId>
-          </exclusion>
-        </exclusions>
+        <version>0.16</version>
```

Comment:
Since ORC-609, we use `0.16`.
- https://github.com/apache/orc/commit/0921709292a7c223491a4a23ef1a25df0de24739#diff-d8225ebcfc11b480a6e4f54e183b67c3ead51635a167c106d928c2abf1f9ef66